06_predict_survival/nbconverted/plot_survival_results.py

```diff
@@ -115,75 +126,77 @@
 # In[7]:
 
 
-results_df['identifier'] = (results_df.cancer_type + '_' +
-                            results_df.training_data)
-results_df.head()
+me_results_df['identifier'] = (me_results_df.cancer_type + '_' +
+                               me_results_df.training_data)
+me_results_df.head()
 
 
 # In[8]:
 
 
-compare_df = au.compare_control_ind(results_df,
-                                    identifier='identifier',
-                                    metric='cindex',
-                                    verbose=True)
-compare_df['cancer_type'] = compare_df.identifier.str.split('_', 1, expand=True)[0]
-compare_df['training_data'] = compare_df.identifier.str.split('_', 1, expand=True)[1]
+me_compare_df = au.compare_control_ind(me_results_df,
+                                       identifier='identifier',
+                                       metric='cindex',
+                                       verbose=True)
+me_compare_df['cancer_type'] = me_compare_df.identifier.str.split('_', 1, expand=True)[0]
+me_compare_df['training_data'] = me_compare_df.identifier.str.split('_', 1, expand=True)[1]
 
-print(len(compare_df))
-compare_df.head()
+print(len(me_compare_df))
+me_compare_df.head()
 
 
 # In[9]:
 
 
 sns.set({'figure.figsize': (18, 8)})
     
-compare_df.sort_values(by='cancer_type', inplace=True)
+me_compare_df.sort_values(by='cancer_type', inplace=True)
 
-sns.boxplot(data=compare_df[~compare_df.cancer_type.isin(drop_cancer_types)],
+sns.boxplot(data=me_compare_df[~me_compare_df.cancer_type.isin(drop_cancer_types)],
```

Comment:
Are there enough samples for a violin plot? Probably not with this many cancer types

Reply:
Yeah, there's only 8 samples per box. I looked at a violin plot here and it doesn't seem to give too much additional information.